src/app/components/Fleet/FleetDevicesTab.tsx

```diff
@@ -14,7 +14,7 @@ const FleetDevicesTab = ({
   onDevicesLoaded: (deviceCount: number) => void;
 }) => {
   const [deviceList, loadingDevices, devicesError, refetch] = useFetchPeriodically<DeviceList>({
-    endpoint: `devices?owner=${fleetName}`,
+    endpoint: `devices?owner=Fleet/${fleetName}`,
```

Comment:
shouldn't it be lowercase ? 
```suggestion
    endpoint: `devices?owner=fleet/${fleetName}`,
```

Reply:
It's a bit confusing, but I think `Fleet` is correct.

The "FleetKind" I think is what's defined here: https://github.com/flightctl/flightctl/blob/main/internal/store/model/fleet.go#L13
But there's another lowercased "FleetKind" for the CLI commands in https://github.com/flightctl/flightctl/blob/main/internal/cli/helpers.go#L14.

On the demo environment, this seems to be confirmed. For the GET fleets response, `kind` equals `Fleet`  and for the `owner` field, you can see it populated for fleets, and it's like this:

```
      "metadata": {
        "creationTimestamp": "2024-02-28T10:22:52Z",
        "generation": 1,
        "labels": {},
        "name": "mlops-pins-fleet",
        "owner": "ResourceSync/pins-demo-resourcesync"
      },
```